runtime/gc_glue_java/MixedObjectScanner.hpp

```diff
@@ -34,42 +34,25 @@
 #include "objectdescription.h"
 #include "GCExtensions.hpp"
 #include "ObjectScanner.hpp"
+#include "HeadlessMixedObjectScanner.hpp"
+
+#include <stdio.h>
 
 /**
  * This class is used to iterate over the slots of a Java object.
  */
-class GC_MixedObjectScanner : public GC_ObjectScanner
+class GC_MixedObjectScanner : public GC_HeadlessMixedObjectScanner
 {
-	/* Data Members */
-private:
```

Comment:
We should keep labels `private: / protected: / public:` for data members, even content is empty. Also please keep `/* Data Members */` comment as well

Reply:
Fixed